pymeasure/display/widgets.py

```diff
@@ -418,3 +422,352 @@ def update_plot(self, filename):
                 new_item = QtGui.QTreeWidgetItem([param.name, str(param)])
                 self.preview_param.addTopLevelItem(new_item)
             self.preview_param.sortItems(0, QtCore.Qt.AscendingOrder)
+
+
+SAFE_FUNCTIONS = {
```

Comment:
It would be great to give a clear description of how to use these in the documentation.